process/transaction/process_test.go

```diff
@@ -397,8 +397,6 @@ func TestTxProcessor_GetSameAccountShouldWork(t *testing.T) {
 //------- checkTxValues
 
 func TestTxProcessor_CheckTxValuesHigherNonceShouldErr(t *testing.T) {
-	t.Skip()
-
 	adr1 := mock.NewAddressMock([]byte{65})
```

Comment:
add t.Parallel, for the 2 other tests as well.

Reply:
Added t.Parallel on the entire test file